backend/src/LibBackend/SqlCompiler.fs

```diff
@@ -445,6 +445,11 @@ let rec lambdaToSql
         // TYPESCLEANUP use args
         | TCustomType (typeName, args) ->
           match Map.get typeName types with
+          // TODO: Deal with alias of record type
+          | Some (CustomType.Alias _) ->
+            error2
+              "The datastore's type is not a record"
+              (FQTypeName.toString typeName)
```

Comment:
Let's add a task around this in "backend work to be done" in notion

Reply:
[Done](https://www.notion.so/darklang/backend-work-to-be-done-0c191de14aed45d8892a3d289d0b5885?pvs=4#76375858b1e4496e86d752dfe8966cde)